src/app/beer_garden/router.py

```diff
@@ -380,13 +378,6 @@ def invalid_source_check(operation: Operation):
             Garden(name=operation.source_garden_name)
         )
 
-        if loaded_garden.status == "NOT_CONFIGURED":
-            logger.error(
-                f"There is no configuration file for {operation.source_garden_name}, "
-                "please validate your children directory for the correct file name"
-            )
-            return True
-
```

Comment:
We should still do this check, but instead look at the connection status for "CONFIGURATION_ERROR"